pkg/render/manager.go

```diff
@@ -94,16 +95,26 @@ func Manager(
 	clusterDomain string,
 	esLicenseType ElasticsearchLicenseType,
 ) (Component, error) {
-	tlsSecrets := []*corev1.Secret{tlsKeyPair}
-	tlsSecrets = append(tlsSecrets, secret.CopyToNamespace(ManagerNamespace, tlsKeyPair)...)
-	tlsAnnotations := make(map[string]string)
+	var tlsSecrets []*corev1.Secret
+	tlsAnnotations := map[string]string{
+		KibanaTLSHashAnnotation: rmeta.SecretsAnnotationHash(kibanaSecrets...),
+	}
+	// If the TLS Secret was passed it means we want to use it, regardless of the certificate management feature.
+	useCertificateManagement := tlsKeyPair == nil && installation.CertificateManagement != nil
```

Comment:
I don't think we should get here if both tlsKeyPair and installation.CertificateManagement are non-nil?

Reply:
If the user brings its own manager-tls, we will use it. For now that is their only way to add extra SANs to the secret. Currently, adding extra SANs is not yet part of the spec.

If the manager-tls is not user-provided while certificatemanagement is enabled, the tlsKeyPair variable will be nil.

So in any case that the manager-tls is provided, we want to use it.

Reply:
I understand why this is so I'll resolve this comment but expect further discussion in my related comment on the manager_controller